server/routerlicious/packages/services-shared/src/index.ts

```diff
@@ -36,3 +36,5 @@ export {
 } from "./webServer";
 export { WholeSummaryReadGitManager } from "./wholeSummaryReadGitManager";
 export { WholeSummaryWriteGitManager } from "./wholeSummaryWriteGitManager";
+export { StartupChecker } from "./startupChecker";
```

Comment:
I think we had talked about not exporting this at all. If I misunderstood, let me know

Reply:
Oops yes I missed changing this. Thanks, will update the `index.ts`